deploy/olm-catalog/kubevirt-hyperconverged/0.0.2/mro.crd.yaml

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
This will get wiped out when the csv gets regenerated.  Edit ` templates/crds/mro.crd.yaml.in` and put `{{if .Converged}}` on the same line as `---`

Reply:
Ok! Done.